crest/Assets/Crest/Crest/Scripts/Shapes/OceanWaveSpectrum.cs

```diff
@@ -444,6 +468,16 @@ public override void OnInspectorGUI()
                 }
             }
 
+            // Display a notice if its being edited as a standalone asset (not embedded in a component) because
+            // it displays the FFT-interface.
+            if (_hostComponentType == null)
+            {
+                EditorGUILayout.Space();
+                EditorGUILayout.HelpBox("This editor is displaying the FFT spectrum settings. " +
+                    "To edit settings specific to the ShapeGerstner component, assign this asset to a ShapeGerstner component " +
+                    "and edit it there by expanding the Spectrum field.", MessageType.Info);
+            }
+
```

Comment:
This would make more sense at the top of the inspector which would be consistent with "This is a shared asset." message when embedded.

Reply:
done

Reply:
Thanks